snscrape/modules/twitter.py

```diff
@@ -317,6 +323,34 @@ def _tweet_to_tweet(self, tweet, obj):
 		kwargs['conversationId'] = tweet['conversation_id'] if 'conversation_id' in tweet else int(tweet['conversation_id_str'])
 		kwargs['lang'] = tweet['lang']
 		kwargs['source'] = tweet['source']
+                
+		# https://developer.twitter.com/en/docs/tutorials/filtering-tweets-by-location								     
+
+		coordinates = None						     
```

Comment:
Please set `kwargs['coordinates']` directly in the blocks below instead of this intermediate variable to avoid duplicating the default value of `None`.